manifests/config.pp

```diff
@@ -20,26 +20,10 @@
     }
   }
 
-  $_db_dirs = flatten($gdm::dconf_hash.keys.map |$profile| {
-    ["/etc/dconf/db/${profile}.d", "/etc/dconf/db/${profile}.d/locks"]
-  })
-
-  ensure_resource('file', $_db_dirs, {
-    ensure  => 'directory',
-    owner   => 'root',
-    group   => 'root',
-    mode    => '0644',
-    recurse => true,
-    purge   => true
-  })
-
   $gdm::dconf_hash.each |String $profile_name, Hash $profiles| {
-    $profiles.each |String $path, Hash $settings| {
-      gnome::dconf { "${profile_name} ${path}":
-        path          => $path,
-        profile       => $profile_name,
-        settings_hash => $settings
-      }
+    dconf::settings { 'GDM Dconf Settings':
```

Comment:
Won't this try to create the same resource more than once?

Reply:
@jeannegreulich Yup, you're 100% correct.